docker/ftest/src/test/resources/tomcat/Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM tutum/tomcat:7.0
-
+FROM tomcat:jre8
```

Comment:
This might be another option. With just `tomcat:jre8` I don't even know which tomcat it is - I would assume latest. So how about locking it on a version which we know our stuff works with? e.g. `tomcat:8.5.11-jre8` (just an example - not sure if that's the right version).